.circleci/continue_config.yml

```diff
@@ -128,13 +128,19 @@ commands:
           command: choco install make
 
   win_setup_requirements:
+    parameters:
+      python_version:
+        type: string
     steps:
+      - restore_cache:
+          name: Restore package cache
+          key: kedro-deps-v1-win-{{ checksum "requirements.txt" }}-{{ checksum "test_requirements.txt" }}
+      - restore_cache:
+          name: Restore conda environment cache
+          key: kedro-deps-v1-win-<<parameters.python_version>>-{{ checksum "requirements.txt" }}-{{ checksum "test_requirements.txt" }}
       - run:
-          name: Install GDAL
-          command: conda activate kedro_builder; conda install -c conda-forge gdal -y
-      - run:
-          name: Install Fiona
-          command: conda activate kedro_builder; conda install -c conda-forge fiona -y
+          name: Install GDAL and Fiona
+          command: conda activate kedro_builder; conda install gdal fiona -c conda-forge -y
```

Comment:
This isn't just tidying but actually makes things ~30s faster, since `conda install fiona` was downgrading some of the dependencies installed in the previous `conda install gdal` step. Installing them both at the same time is better.

Reply:
Nice catch! Would be nice to add an inline comment about that, completely optional though.